examples/example_slice_explorer.py

```diff
@@ -1,7 +1,6 @@
 from argparse import ArgumentParser, BooleanOptionalAction
 
-from pan3d import DatasetBuilder
-from pan3d import SliceExplorer
+from pan3d.explorers.slicer import XArraySlicer
```

Comment:
What is the point of that file? You can run it from the cli by doing 

`xr-slicer` or `python -m  pan3d.explorers.slicer`